src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -134,7 +134,9 @@ export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Va
   @HostListener('blur')
   public onBlur(): void {
     const inputValue = this.getInputValue();
-    const numericValue = inputValue ? this.autonumericInstance.getNumber() : undefined;
+    const currencySymbol = (<{ [key: string]: any; }>this.autonumericOptions)['currencySymbol'];
```

Comment:
Would you mind slapping a comment on here to describe what's going on? It's a little hard to follow.